funding/manager_test.go

```diff
@@ -4429,8 +4449,36 @@ func TestFundingManagerZeroConf(t *testing.T) {
 		Tx: fundingTx,
 	}
 
+	// After 6 confirmation a channel which should be announced to the
+	// network will be readded to the network graph and announced by its
+	// real ShortChannelID. In case of a public zeroconf channel we need
+	// to make sure the MessageFlags of the prior edgeinfo are not copied
+	// but redetermined.
+	//nolint:lll
+	alice.fundingMgr.cfg.DeleteAliasEdge = func(scid lnwire.ShortChannelID) (*channeldb.ChannelEdgePolicy, error) {
+		//nolint:lll
+		return &channeldb.ChannelEdgePolicy{
+			MessageFlags:              lnwire.ChanUpdateRequiredMaxHtlc ^ lnwire.ChanUpdateDontForward,
```

Comment:
Nit: `xor` seems like a strange choice to me for unioning bit vectors.

In theory this ought not to matter since they should be independent bits anyway, meaning identical semantics for `or` and `xor`, but in the odd case where it's wrong the behavior may be highly unpredictable. I'm assuming there's precedent for this in the codebase though? Probably not an issue, but would love to hear if there's some wisdom in doing this that is nonobvious.

Reply:
I agree with you and will change it to `or` which is the safer choice here. I did not find some other example in the codebase, thanks